configs/edison/config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison

```diff
@@ -131,6 +131,7 @@ polarPlot = False
 [streamfunctionMOC]
 ## options related to plotting the streamfunction of the meridional overturning 
 ## circulation (MOC)
+maxChunkSize = 1000
```

Comment:
This was the suggestion yesterday if I understood correctly.  Having these climatologies at different scales, e.g., monthly, yearly, 5-yr, 10-yr, etc also has scientific benefit.  

For now we need to get past this memory error but this is just something to keep in mind.